disable_odoo_online/data/ir_cron.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="mail.ir_cron_module_update_notification" model="ir.cron">
+        <field name="active" eval="True" />
```

Comment:
OK let's say (as @lasley said) that active should be True. But then my question is: why do we keep this line? What about removing the entire `ir_cron.xml` file?

Reply:
Looks to me like this logic was here before we realized that we were not allowed to disable the update notification for EE. Tracing it [into the code](https://github.com/odoo/odoo/blob/10128fa7e8c743e9c838743204fb8116bcdeefd6/addons/mail/data/mail_data.xml#L17), this cron does call `update_notification`. 

In this light, I think we've broken contract for anyone that has this installed on EE

Reply:
Example: if someone decides to disable manually this cron for some reason, we are forcing it to set to active=True again as soon as he installs this module (or makes an update all, for example). There is not even set <odoo noupdate="1"> in the actual solution. It's still not clear to me, sorry.

Reply:
@hbrunn - you wrote this one, do you remember?

Reply:
my version of this was disabling it: https://github.com/OCA/server-tools/blob/10.0/disable_odoo_online/data/ir_cron.xml#L5 - so yes, whoever noticed this shouldn't be done any more should have just ditched the file. check the [commit history](https://github.com/hieulucky111/server-brand/commit/9492ca0fa7fd67b043eb1a3d1ed22003003d7498#diff-2b288a3e41920769298009277a3d43e2). speaking of which, you shouldn't approve PRs which don't take the [commit history](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#technical-method-to-migrate-a-module-from-100-to-110-branch) with them, especially for cases like the current one, it's utterly annoying having to puzzle together the evolution of a module/file if the link is cut

Reply:
Clear, thank you!
@hieulucky111 can you please remove the ir_cron.xml file?